apps/cowswap-frontend/src/modules/ordersTable/pure/ReceiptModal/index.tsx

```diff
@@ -191,20 +194,20 @@ export function ReceiptModal({
               </styledEl.Field>
             )}
 
+            <styledEl.Field>
+              <FieldLabel label="Filled" tooltip={twapOrder ? tooltips.FILLED_TWAP : tooltips.FILLED} />
+              <FilledField order={order} />
+            </styledEl.Field>
```

Comment:
aha, ok, i see. Cool i thought we had it already! then it does make a lot of sense